ci-operator/step-registry/gather/alibabacloud-console/gather-alibabacloud-console-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export ALIBABA_CLOUD_CREDENTIALS_FILE=${SHARED_DIR}/inicreds
+export ALIBABA_CLI_CREDENTIALS_FILE=${SHARED_DIR}/config
+
+wget https://aliyuncli.alicdn.com/aliyun-cli-linux-latest-amd64.tgz -O /tmp/aliyun-cli.tgz
+pushd /tmp
```

Comment:
nit: you can save yourself the `/tmp/` in the `wget` line above by shifting it down below this `pushd`:

```sh
pushd /tmp
wget https://aliyuncli.alicdn.com/aliyun-cli-linux-latest-amd64.tgz aliyun-cli.tgz
```

Also, if you feel jumpy about randomly pulling down binaries, [this][1] is an example of hashing to confirm that you're getting what you expected to get.

[1]: https://github.com/openshift/release/blob/6a54b9833d4ff6fec77f90dcfb5d7aa174fb8203/ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-commands.sh#L7-L13

Reply:
I didn't see any hash file provided during the download process. I have filed an issue here https://github.com/aliyun/aliyun-cli/issues/401.

We can revisit.

I'll remove the tmp from the wget.